configuration.md

```diff
@@ -14,10 +14,12 @@ Casual supports the following configuration options which can be set using envir
 * `CASUAL_CONFIG_FILE` - Set path for casual configuration file. If not provided, default are used.
 * `CASUAL_INBOUND_STARTUP_MODE` - Set mode for inbound startup. Default `immediate`. Alternatives `trigger`, `discover`.
     See [Inbound Startup Configuration](inbound.md#startup-configuration) for more details.
-* `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
-* `CASUAL_INBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
+* `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO - deprecated, please see `CASUAL_OUTBOUND_USE_EPOLL` instead.
+* `CASUAL_INBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO - deprecated, please see `CASUAL_OUTBOUND_USE_EPOLL` instead.
+* `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO, this goes for outbound, inbound and reverse inbound.
```

Comment:
This should say something else than `CASUAL_OUTBOUND_USE_EPOLL`? `CASUAL_USE_EPOLL` maybe based on the code changes. Also goes for the deprecation notices which seem to reference the incorrect env

Reply:
It is indeed CASUAL_USE_EPOLL, documentation was not updated correctly - it is now.

Reply:
The commented line (19) should also be updated to `CASUAL_USE_EPOLL` instead of `CASUAL_OUTBOUND_USE_EPOLL`, right?

Reply:
Fixed